src/apis/data/endpoints.py

```diff
@@ -0,0 +1,42 @@
+from flask_restplus import Namespace, Resource
+from apis.data.services import ReportService
+
+my_api = Namespace('data', description='Data related operations')
+
+
+@my_api.route('/foo_bar')
```

Comment:
Se puder, ja pode remover esse EP que era apenas um exemplo no início do projeto.

Reply:
Beleza!